cppwamp/include/cppwamp/internal/client.hpp

```diff
@@ -673,6 +673,21 @@ class Client : public ClientInterface, public Peer<TCodec, TTransport>
             {
                 yield(reqId, std::move(error));
             }
+
+            // TODO: Workaround to prevent aborting when
```

Comment:
We will make this a feature of CppWAMP, so it's not really a workaround. Please change comment to "Forward Variant conversion exceptions as ERROR messages". Add these same catch clauses in `dispatchEvent` above, except that they will log about the failed conversion.